### DISCUSSION TITLE
Multiple IPs for Manager Interface

### DISCUSSION THREAD

**Discussion by pmzhou** [Security-Onion-Solutions/securityonion#6929] [Unsupported Versions] [answered]
Is there a way to use **multiple IP addresses on the manager interface**? I am trying to reach my SO from outside my LAN using Zerotier IP. [ZeroTier](https://www.zerotier.com/) is some kind of a "network hypervisor." It creates a virtual network. 

Scenario:
LAN - 192.168.0.40
Zerotier 10.10.0.40

I can reach 10.10.0.40 but it redirects to 192.168.0.40 which makes it impossible to access from outside the network.
I need both IPs to be associated with the the managerip. The current documentation only show how to change the current IP to some other (unless if I missed it somehow). I hope this is doable.

**Comment by dougburks** [comment, marked as answer]
One option might be to use a FQDN that resolves to different IP addresses depending on where you are. If that's an option, then just set `url_base` to that FQDN:
https://docs.securityonion.net/en/2.3/url-base.html
